docs/geti_tune/pipeline.md

```diff
@@ -69,10 +82,10 @@ Each sink is identified by a unique id, and optionally a friendly name chosen by
 
 ### Models
 
-Geti Tune supports models exported from Geti in OpenVINO IR format (.xml + .bin). All tasks are supported, including
-classification, detection, segmentation, anomaly and keypoint detection.
+A pipeline performs inference using models OpenVINO IR format (.xml + .bin), converted after fine-tuning.
+The supported task types include image classification, object detection and instance segmentation.
```

Comment:
This can be probably omitted because it duplicates the section from labels.md:

> In Geti Tune, every _project_ addresses a specific _task_. The currently supported task types are:
> 
> - **Classification**: Categorize images into one (_multiclass_) or more (_multilabel_) classes.
> - **Detection**: Identify and localize objects in images using bounding boxes.
> - **Instance Segmentation**: Identify and segment objects in images using polygons.